sdk/core/amqp-common/src/errors.ts

```diff
@@ -518,8 +519,24 @@ export function isSystemError(err: any): boolean {
     err.code &&
     typeof err.code === "string" &&
     (err.syscall && typeof err.syscall === "string") &&
-    (err.errno &&
-      (typeof err.errno === "string" || typeof err.errno === "number"))
+    (err.errno && (typeof err.errno === "string" || typeof err.errno === "number"))
+  ) {
+    result = true;
+  }
+  return result;
+}
+
+/**
+ * Utility to check if given object is an instance of a browser based WebSocket error
+ * @param err object that may contain error information
+ */
+export function isWebsocketError(err: any): boolean {
+  let result: boolean = false;
+  if (
+    !isNode &&
+    typeof window !== undefined &&
+    err.type === "error" &&
+    err.target instanceof (window as any).WebSocket
```

Comment:
Since Websockets can be used in a non browser scenario, I would suggest that the name `isWebsocketError` be updated to indicate that this is a browser specific scenario.